resources/aws/environment.go

```diff
@@ -166,6 +166,10 @@ func (e *Environment) DefaultShutdownBehavior() string {
 	return e.GetStringWithDefault(e.InfraConfig, DDInfraDefaultShutdownBehavior, e.envDefault.ddInfra.defaultShutdownBehavior)
 }
 
+func (e *Environment) DefaultFargateLoadBalancer() bool {
+	return e.GetBoolWithDefault(e.InfraConfig, DDInfraEcsFargateCapacityProvider, e.envDefault.ddInfra.defaultFargateLoadBalancer)
+}
+
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
This should be defined in `common/config/environments.go` as `InfraShouldDeployFakeintakeWithLB`